lib/experiment/local/client.rb

```diff
@@ -38,17 +37,37 @@ def initialize(api_key, config = nil)
     #
     # @return [Hash[String, Variant]] The evaluated variants
     def evaluate(user, flag_keys = [])
+      warn 'evaluate is deprecated, please use evaluate_v2 instead.'
```

Comment:
deprecation notices should always be IDE/compile side, not printing to stderr/stdout